src/Tribe/Service_Provider.php

```diff
@@ -44,8 +44,11 @@ public function register() {
 		$this->container->singleton( 'tickets.editor', 'Tribe__Tickets__Editor', array( 'hook' ) );
 
 		$this->container->singleton( 'tickets.admin.notices', 'Tribe__Tickets__Admin__Notices', array( 'hook' ) );
-		// Promoter Observer
-		$this->container->singleton( 'tickets.promoter.observer', 'Tribe__Tickets__Promoter_Observer', array( 'hook' ) );
+
+		// Promoter
+		$this->container->singleton( 'tickets.promoter.integration', 'Tribe__Tickets__Promoter__Integration', array( 'hook' ) );
```

Comment:
We should eventually figure out a Provider for Promoter.

Reply:
We can separate all the providers from this super class soon, for now I'll keep it since we're too close for release.